features.html

```diff
@@ -30,58 +29,61 @@ <h2>Features</h2>
-      see all of the books that they have written in the library.
-    </quote>
-
     <quote>
       <b>Persistent URLs</b>
-      Allows the user to bookmark their queries or records to allow permanent access to a page they were once on.
+      Search and record pages never change their URLs, allowing users to easily bookmark and share queries
+      and other useful content.
     </quote>
 
     <quote>
-      <b>Zotero Compatible</b>
+      <b>Compatible with Reference Management Tools</b>
       Your users can save and tag any records with
-      <a href="http://www.zotero.org/">Zotero</a> or any other
-      <a href="http://ocoins.info/">COinS</a> based application so they
-      can store their records in one place.
+      <a href="http://www.zotero.org/">Zotero</a> and other commonly used reference management tools.
+      Data is embedded using the <a href="https://en.wikipedia.org/wiki/COinS">COinS</a> standard and
+      can also be <a href="https://vufind.org/wiki/development:howtos:supporting_a_new_export_format">exported</a>
+      in a variety of ways.
     </quote>
 
     <quote>
       <b>Internationalization</b>
-      The interface has translations available in Arabic, Brazilian Portuguese, Chinese,
-      Dutch, English, French, German, Japanese, Spanish and many more. And if you
-      want to create your own translation - it's easy.  You can even create a new English
-      translation if you want to change some of the wording used in the interface -
-      it makes the UI that much easier to customize.
+      The interface has translations available in more than thirty languages, including
+      Arabic, Brazilian Portuguese, Chinese, Dutch, English, French, German, Japanese and Spanish.
+      <a href="https://vufind.org/wiki/development:architecture:localization">Supporting new languages</a> is easy,
+      and you can also override translations to adjust the user interface without changing any templates or code.
     </quote>
 
     <quote>
-      <b>Access Your Data: Open Search, OAI, Solr</b>
+      <b>Access Your Data: OAI-PMH, APIs, Solr</b>
       VuFind® has many APIs to interact with the search, data and many other features.
-      You can syndicate your record data with other institutions via an OAI server.
-      You can search using VuFind®'s algorithms via OpenSearch.
-      And if you want complete access to your indexed data, you can interact with Solr,
+      You can syndicate your record data with other institutions via an
+      <a href="https://vufind.org/wiki/indexing:tracking_record_changes#oai-pmh_server_functionality">OAI server</a>.
+      You can search using VuFind®'s algorithms via its <a href="https://vufind.org/wiki/development:apis">built-in APIs</a>.
+      And if you want complete access to your indexed data, you can interact directly with Solr,
       VuFind®'s backend search and index engine.
     </quote>
+
+    <quote>
+      <b>Integration with Content Providers</b>
```

Comment:
I don't think highlighting author biographies specifically makes a lot of sense, so I generalized it and moved it down here.

Reply:
Good idea -- edit made.